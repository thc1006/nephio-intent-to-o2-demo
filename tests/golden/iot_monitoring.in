Create IoT monitoring infrastructure with high capacity for industrial sensors across both edge sites