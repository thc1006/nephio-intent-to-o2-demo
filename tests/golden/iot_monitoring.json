{
  "intentId": "intent_iot_monitoring",
  "name": "IoT Monitoring Infrastructure",
  "description": "Create IoT monitoring infrastructure with high capacity for industrial sensors across both edge sites",
  "targetSite": "both",
  "parameters": {
    "type": "iot_infrastructure",
    "requirements": {
      "capacity": "100000 devices",
      "bandwidth": "10Gbps",
      "reliability": "99.9%"
    },
    "configuration": {
      "service": "iot_monitoring",
      "protocol": "mqtt",
      "deviceType": "industrial_sensors"
    }
  },
  "priority": "medium",
  "lifecycle": "draft"
}